.editorconfig

```diff
@@ -0,0 +1,27 @@
+# editorconfig.org
+
+# This is the project's root directory
```

Comment:
Please remove comments, it's cleaner and nobody looks or modifies this file anyway.